concepts/strings/introduction.md

```diff
@@ -1,5 +1,42 @@
 # Introduction
 
-TODO: the content below is copied from the exercise introduction and probably needs rewriting to a proper concept introduction
+A `string` in C++ is a mutable object that represents text as a sequence of Unicode characters (letters, digits, punctuation, etc.). 
```

Comment:
I'd expect someone that is trying to learn C++ to know what Unicode is but what if someone doesn't? Shall we maybe add in the `links.json` an explanation of what Unicode is?

Reply:
Or take it out? Because to have real Unicode compatibility we might need to talk about the mess behind the different boundaries https://stackoverflow.com/a/50407375/4919081

Reply:
Yeah, taking it out works too.